app/models/benefit_coverage_period.rb

```diff
@@ -255,11 +259,14 @@ def all
       organizations = Organization.exists(:"hbx_profile.benefit_sponsorship.benefit_coverage_periods" => true)
       organizations.size > 0 ? organizations.first.hbx_profile.benefit_sponsorship.benefit_coverage_periods : nil
     end
-
   end
 
   private
 
+  def over_thirty?(family_members)
+    family_members.detect{|family_member| DateTime.now - 30.years >= family_member.person.dob}.present?
```

Comment:
Please use enrollment effective on as a basis for calculating age. 

Also reuse age_on method from person model